src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/Type.cs

```diff
@@ -585,16 +585,6 @@ public PredefinedType getPredefType()
             return getAggregate().GetPredefType();
         }
 
-        ////////////////////////////////////////////////////////////////////////////////
-        // Is this type System.TypedReference or System.ArgIterator?
-        // (used for errors because these types can't go certain places)
-
-        public bool isSpecialByRefType()
-        {
-            // ArgIterator, TypedReference and RuntimeArgumentHandle are not supported.
```

Comment:
They still can't be used with dynamic as they can't be cast to object. The desktop version checks for the relevant enum values that came from the porting of the pre-roslyn compiler, but at no point would those values ever be returned.